builtin/v16/migration/top.go

```diff
@@ -90,6 +98,20 @@ func MigrateStateTree(ctx context.Context, store cbor.IpldStore, newManifestCID
 		return cid.Undef, xerrors.Errorf("incomplete migration specification with %d code CIDs, need %d", len(migrations)+len(deferredCodeIDs), len(oldManifestData.Entries))
 	}
 
+	// The Evm Actor
+
+	newEvmCodeCID, ok := newManifest.Get(manifest.EvmKey)
+	if !ok {
+		return cid.Undef, xerrors.Errorf("code cid for system actor not found in new manifest")
+	}
+
+	migrations[evm15Cid] = migration.CachedMigration(cache, evmMigrator{newEvmCodeCID})
+
+	if len(migrations)+len(deferredCodeIDs) != len(oldManifestData.Entries) {
+		return cid.Undef, xerrors.Errorf("incomplete migration specification with %d code CIDs, need %d", len(migrations)+len(deferredCodeIDs), len(oldManifestData.Entries))
+	}
```

Comment:
this block is a dupe of lines 97-99, this is the right place for it so just remove the dupe above; ew

Reply:
Thank you! ed2876d